src/main/java/Mona/Task.java

```diff
@@ -0,0 +1,36 @@
+package mona;
+public class Task {
```

Comment:
Since your task types inherit from Task class, perhaps you could consider making Task into its own package?